File: easy/js/squareEachNum.js
function squareEachNum(num) {
  if (num < 10) {
    return num;
  }
  return +num
    .toString()
    .split('')
    .map((num) => num * num)
    .join('');
}
console.log(squareEachNum(25)); //	425
// O(n)

File: easy/dart/squareEachNum.dart
class SquareEachNum {
  static num result(num n) {
    if (n < 10) {
      return 0;
    }
    return num.parse(n
        .toString()
        .split('')
        .map((String strNum) => num.parse(strNum) * num.parse(strNum))
        .join(''));
  }
}
